caravel/utils.py

```diff
@@ -310,6 +310,8 @@ def json_int_dttm_ser(obj):
     val = base_json_conv(obj)
     if val is not None:
         return val
+    if isinstance(obj, date):
+        obj = datetime.combine(obj, datetime.min.time())
     if isinstance(obj, datetime):
```

Comment:
Fix transforms data before serializing (date -> datetime)
Do you prefer **elif** way ?